lib/web_ui/test/canvas_test.dart

```diff
@@ -76,5 +77,39 @@ void testMain() {
       expect(mockCanvas.methodCallLog[0].methodName, 'clear');
       expect(mockCanvas.methodCallLog[1].methodName, 'endOfPaint');
     });
+
+    testCanvas('can reuse a paragraph builder',
```

Comment:
Consider also adding a screenshot test. Now that we have Skia Gold, the process is much simpler.

For CanvasKit, a good place would be https://github.com/flutter/engine/blob/main/lib/web_ui/test/canvaskit/canvas_golden_test.dart.

For HTML, https://github.com/flutter/engine/blob/main/lib/web_ui/test/html/bitmap_canvas_golden_test.dart

Reply:
Nice, I was not aware we had Skia Gold for this repo.